### PR TITLE
Issue 1 create an entry

### PR DESCRIPTION
**Pull request by gregg-garceau**
https://user-images.githubusercontent.com/88458577/132556868-ec226495-72c1-4ff3-a73f-382109dd5c4f.mp4

### CODE REVIEW

File: js/data.js [hunk 1]
```diff
@@ -6,3 +6,10 @@ var data = {
   editing: null,
   nextEntryId: 1
 };
+
+window.addEventListener('beforeunload', function (event) {
+  var dataJson = JSON.stringify(data);
+  var entriesJson = JSON.stringify(data.entries);
+  localStorage.setItem('data', dataJson);
+  localStorage.setItem('entries', entriesJson);
```
Comment (scbowler) [inline js/data.js]: Your entries should be stored on the `entries` property of the `data` object. Everything that is stored should be within the `data` object so you only have to save one thing to local storage.

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, just to cleanup the `beforeunload` event handler callback. Let me know if you have any questions.

File: js/data.js (hunk 1 above)
Comment (gregg-garceau) [reply js/data.js]: https://user-images.githubusercontent.com/88458577/132558725-d9c47710-0610-453c-9630-23e927358101.mov

Comment (gregg-garceau) [reply js/data.js]: Fixed

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**